src/types/response/futures.ts

```diff
@@ -32,21 +32,30 @@ export interface FuturesAccount {
 }
 
 export interface FuturesSymbolRule {
-  baseCoin: string;
-  buyLimitPriceRatio: string;
-  feeRateUpRatio: string;
-  makerFeeRate: string;
-  minTradeNum: string;
-  openCostUpRatio: string;
-  priceEndStep: string;
-  pricePlace: string;
-  quoteCoin: string;
-  sellLimitPriceRatio: string;
-  sizeMultiplier: string;
-  supportMarginCoins: string[];
-  symbol: string;
-  takerFeeRate: string;
-  volumePlace: string;
+  baseCoin: string
+  buyLimitPriceRatio: string
+  feeRateUpRatio: string
+  limitOpenTime: string
+  maintainTime: string
+  makerFeeRate: string
+  maxOrderNum: string
+  maxPositionNum: string
+  minTradeNum: string
+  minTradeUSDT: string
+  offTime: string
+  openCostUpRatio: string
+  priceEndStep: string
+  pricePlace: string
+  quoteCoin: string
+  sellLimitPriceRatio: string
+  sizeMultiplier: string
+  supportMarginCoins: string[]
+  symbol: string
+  symbolName: string
+  symbolStatus: string
+  symbolType: string
+  takerFeeRate: string
+  volumePlace: string
 }
```

Comment:
Nitpick (sorry), semicolons missing. You can run the linter and it'll add them back automatically (as well as other misc formatting things)